static/src/js/components/AdminRetrievals/__tests__/AdminRetrievals.test.jsx

```diff
@@ -1,40 +1,74 @@
+import { screen } from '@testing-library/react'
 import React from 'react'
-import Enzyme, { shallow } from 'enzyme'
-import Adapter from '@wojtekmaj/enzyme-adapter-react-17'
-
+import setupTest from '../../../../../../jestConfigs/setupTest'
 import { AdminRetrievals } from '../AdminRetrievals'
-import { AdminPage } from '../../AdminPage/AdminPage'
 import { AdminRetrievalsList } from '../AdminRetrievalsList'
+import { AdminRetrievalsForm } from '../AdminRetrievalsForm'
+
+jest.mock('../AdminRetrievalsList', () => ({
+  AdminRetrievalsList: jest.fn(() => <div data-testid="admin-retrievals-list" />)
+}))
+
+jest.mock('../AdminRetrievalsForm', () => ({
+  AdminRetrievalsForm: jest.fn(() => <div data-testid="admin-retrievals-form" />)
+}))
 
-Enzyme.configure({ adapter: new Adapter() })
+const mockRetrievals = {
+  allIds: ['retrieval-1', 'retrieval-2'],
+  byId: {
+    'retrieval-1': {
+      id: 'retrieval-1',
+      title: 'Test Retrieval 1'
+    },
+    'retrieval-2': {
+      id: 'retrieval-2',
+      title: 'Test Retrieval 2'
+    }
+  },
+  pagination: {
+    pageNum: 1,
+    pageSize: 20,
+    totalResults: 2
+  },
+  sortKey: 'created_at'
+}
 
-function setup() {
-  const props = {
+const setup = setupTest({
+  Component: AdminRetrievals,
+  withRouter: true,
+  defaultProps: {
     historyPush: jest.fn(),
     onAdminViewRetrieval: jest.fn(),
+    onFetchAdminRetrievals: jest.fn(),
     onUpdateAdminRetrievalsSortKey: jest.fn(),
     onUpdateAdminRetrievalsPageNum: jest.fn(),
-    retrievals: {
-      allIds: [],
-      byId: {},
-      pagination: {},
-      sortKey: ''
-    }
+    retrievals: mockRetrievals
   }
+})
 
-  const enzymeWrapper = shallow(<AdminRetrievals {...props} />)
+describe('when AdminRetrievals is rendered', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
```

Comment:
This shouldn't be necessary, we have a global config in jest.config.cjs that runs clearAllMocks before every test